tests/unit/test_network_of_populations/test_init.py

```diff
@@ -412,7 +412,7 @@ def test_doInternalInfectionProcess_between_ages():
 
     new_infected = np.getInternalInfectiousContactsInNode(current_state, age_matrix, 1.0, ["I", "A"], True,
                                                           numpy.random.default_rng(123))
-    assert new_infected == {"m": 11, "o": 11}
+    assert new_infected == {"m": 12, "o": 13}
```

Comment:
I don't quite understand why this change?  I'm sure I'm just not quite understanding something.

Reply:
The order of operations is now different, so the model results are different.

Reply:
> I believe the numbers were just chosed to match whatever the model returned with the seed 123 previously
Just to confirm: indeed it was